docs/estimators/index.rst

```diff
@@ -64,13 +65,17 @@ Quantity          Definition
 norm_errp		  Positive error of the norm
 norm_errn	      Negative error of the norm
 norm_ul			  Upper limit of the norm
+stat			  Fit statistics value of the best fit hypothesis
+stat_null		  Fit statistics value of the null hypothesis
+================= =================================================
+npred		  	  Predicted counts of the best fit hypothesis
```

Comment:
In the GitHub rendering, these lines are not coming inside the table, but rather in a chaotic manner.

Reply:
Thanks, indeed this was because of mixture of tabs and whitespace. Fixed.